sdk/storage/storage-blob/README.md

```diff
@@ -133,19 +133,52 @@ Below are the snippets to get started with the Azure Storage blob client library
 
 Use the constructor to create a instance of `BlobServiceClient`.
 
-```javascript
-// Enter your storage account name and shared key
-const account = "account";
-const accountKey = "accountkey";
+- **Recommended way to instantiate a `BlobServiceClient` - with `DefaultAzureCredential` from `@azure/identity` package**
+  [Note - ONLY AVAILABLE IN NODE.JS RUNTIME]
```

Comment:
Why Node.js only? One of the AAD OAuth 2.0 protocol scenario is implict access in single page web app.

Reply:
Yeah, the example below is node only but AAD should work in browser.